src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -76,6 +78,10 @@ document.addEventListener('DOMContentLoaded', _e => {
 // Mount react components
 const commonStore = createCommonStore();
 document.addEventListener('DOMContentLoaded', _e => {
+  const crisisModal = document.getElementById('modal-crisisline');
```

Comment:
an ugly hack but faster than refactoring the modal id we're using across 2 sites. when we refactor this, will push to only use classNames.